openlibrary/templates/type/edition/view.html

```diff
@@ -97,6 +97,12 @@
 
 $var title: $title
 
+<style>
+ul:has([href^="/request"]) {
+  display: none !important;
+}
+</style>
+
```

Comment:
I hope this should no longer be necessary 😁 I think it was triggering some weird cause that caused two read button to render on I think the books page?

```suggestion
```